src/utils/fetchclient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing the 'eslint-disable' statement if there's no specific reason to disable the rule. It's better to fix the typing issue and avoid using 'any' type.